apps/next/app/dashboard/create-team/[[...slug]]/page.tsx

```diff
@@ -0,0 +1,5 @@
+import { CreateOrganization } from "@clerk/nextjs";
+
+export default function CreateTeamPage() {
+  return <CreateOrganization afterCreateOrganizationUrl="/dashboard" />;
+}
```

Comment:
I think I'd prefer this to be a modal, since it doesn't really fit into the dashboard as its own page as of now

If Clerk doesn't have a create org modal then it could be a shadcn modal that contains the `<CreateOrganization>` component

Reply:
Done.